recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,39 @@
+c_compiler_version:                       # [unix]
+  - 13                                    # [osx]
+  - 10                                    # [linux]
+cxx_compiler_version:                     # [unix]
+  - 13                                    # [osx]
+  - 10                                    # [linux]
+llvm_openmp:                              # [osx]
+  - 13                                    # [osx]
+python:
+  # part of a zip_keys: python, python_impl, numpy
+  - 3.8.* *_cpython
+python_impl:
+  - cpython
+apr:
+  - '1'
+boost:
+  - 1.74.0
+cfitsio:
+  - 4.0.0
+eigen:
+  - '3.3'
```

Comment:
We need 3.4, actually.

Reply:
Won’t that be the next major version update?

Reply:
Yeah, I guess so.  In theory, it doesn't matter until we actually drop the eups version, but we might as well do them at the same time.